.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,45 @@
+name: Tutorials
+
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - "examples/tutorials/**"
+    - "!examples/tutorials/translations/**"
```

Comment:
This excludes changes to the translations from the diff paths that trigger the English tutorial tests.